themes-default/slim/static/js/store.js

```diff
@@ -0,0 +1,195 @@
+            anonRedirect: null,
+            torrents: null
+        },
+        shows: [],
+        defaults: {
+            show: {
+                id: {
+                    tvdb: null,
+                    slug: null
+                },
+                rating: {
+                    imdb: {
+                        rating: null,
+                        votes: null
+                    }
+                },
+                country_codes: [], // eslint-disable-line camelcase
+                network: null,
+                airs: null,
+                config: {
+                    qualities: null,
+                    defaultEpisodeStatus: null,
+                    dvdOrder: null,
+                    seasonFolders: null,
+                    scene: null,
+                    sports: null,
+                    paused: null,
+                    location: null,
+                    airByDate: null,
+                    release: null,
+                    aliases: null,
+                    subtitlesEnabled: null,
+                    anime: null
+                }
+            }
+        }
+    },
+    mutations: {
+        [LOGIN_PENDING]() {},
+        [LOGIN_SUCCESS]() {},
+        [LOGIN_FAILED]() {},
+        [LOGOUT]() {},
+        [REFRESH_TOKEN]() {},
+        [REMOVE_AUTH_ERROR]() {},
+        [SOCKET_ONOPEN](state) {
+            state.socket.isConnected = true;
+        },
+        [SOCKET_ONCLOSE](state) {
+            state.socket.isConnected = false;
+        },
+        [SOCKET_ONERROR](state, event) {
+            console.error(state, event);
+        },
+        // Default handler called for all socket methods
+        [SOCKET_ONMESSAGE](state, message) {
+            state.socket.message = message;
+        },
+        // Mutations for socket reconnect methods
+        [SOCKET_RECONNECT](state, count) {
+            console.info(state, count);
+        },
+        [SOCKET_RECONNECT_ERROR](state) {
+            state.socket.reconnectError = true;
+        },
+        [ADD_CONFIG](state, config) {
+            state.config = config;
+        },
+        [ADD_SHOW](state, show) {
+            const { shows } = state;
+            const showExists = shows.filter(({ id, indexer }) => id[indexer] === show.id[indexer]).length === 1;
+            if (showExists) {
+                state.shows[shows.indexOf(showExists)] = show;
+            } else {
+                state.shows.push(show);
+            }
+        }
+    },
+    actions: {
+        getConfig(store) {
+            return api.get('/config/main').then(res => {
+                store.commit(ADD_CONFIG, res.data);
+            });
+        },
+        getShow(store, { indexer, id }) {
+            return api.get('/series/' + indexer + id).then(res => {
+                store.commit(ADD_SHOW, res.data);
+            });
+        }
+    },
+    plugins: []
+});
+
+Vue.use(VueNativeSock, 'ws://localhost:8081/x/ws/ui', {
+    store: window.store,
+    format: 'json'
+});
```

Comment:
Do you know if doing this makes it try to connect to the websocket?
Do you need to add `connectManually: true,` to disable that?